client/src/utils/api.ts

```diff
@@ -0,0 +1,278 @@
+import downloadZip from "./downloadZip";
+
+const BASE_URL = 'http://localhost:8080';
+interface CustomResponse {
+	zip?: string;
+	error?: boolean;
+	errorData?: string;
+	errorTitle?: string;
+	message?: string;
+}
+type StatusProps = {
+	message: string;
+	isSuccessful: boolean;
+	isEnd: boolean;
+};
+
+const settings = {
+	notificationLoaded: '.loaded',
```

Comment:
can find usages.... Is these class names?

Reply:
It's from previous PR and solution, yeap forgot to remove.